experimental/LieAlgebras/src/LieAlgebraModule.jl

```diff
@@ -54,21 +59,46 @@ gens(L::LieAlgebraModule{C}) where {C<:RingElement} = basis(L)
 
 gen(L::LieAlgebraModule{C}, i::Int) where {C<:RingElement} = basis(L, i)
 
+@doc raw"""
+    dim(V::LieAlgebraModule{C}) -> Int
+
+Return the dimension of the Lie algebra module `V`.
+"""
 dim(V::LieAlgebraModule{C}) where {C<:RingElement} = V.dim
```

Comment:
Again off-topic, but: you can do away with `C` in many cases, making the code maybe more readable?
```suggestion
    dim(V::LieAlgebraModule) -> Int

Return the dimension of the Lie algebra module `V`.
"""
dim(V::LieAlgebraModule) = V.dim
```

Reply:
Didn't know that. I will look into it for a future PR.